releasenotes/notes/batch-opt-var-generation-63d308b550972f9b.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add the batch variable generation for optimization, i.e.,
+    `QuadraticProgram.{continuous,integer,binary}_var_{dict,list}`.
+    These methods allows users to add multiple variables at once by specifying a number of variables
```

Comment:
```suggestion
    These methods allow users to add multiple variables at once by specifying a number of variables
```

Reply:
Thank you for fixing it.

Reply:
Sure, since the reno content ends up as public facing release notes I know they go through and and fix them up later. While we catch spelling errors it does not fix other aspects which is why I try to remember to read them and fix them up beyond any spelling.